music.js

```diff
@@ -1,17 +1,40 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 }
 
+
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.artist.songs.push(this);
+  this.playCount = 0;
+
+
 
   // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
   // You'll probably want to do that in here somewhere...
 }
 
+Song.prototype.play = function() {
+  this.playCount ++;
```

Comment:
Don't put a space between the property name and the `++` operator. Very nitpicky but it's a style thing :)